controllers/object_controls.go

```diff
@@ -773,6 +783,48 @@ func applyCommonDaemonsetConfig(obj *appsv1.DaemonSet, config *gpuv1.ClusterPoli
 	return nil
 }
 
+// Transform the host-root volume if required for the Daemonset passed as input
+func transformHostRootVolume(obj *appsv1.DaemonSet, hostRoot string) {
```

Comment:
Let's start adding test cases for `object_controls.go`. A test case that covers this new method would be a good start

Reply:
Added test cases for the host root transforms.

Reply:
Thanks!